src/Model/Filter/Boolean.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Search\Model\Filter;
+
+class Boolean extends Base
+{
+
+    /**
+     * Default configuration.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'truthy' => [1, true, '1', 'true',  'yes', 'on'],
```

Comment:
when the constant have the same name as the value, it makes little sense to me :) 

if it was `::POSITIVE = 'truthy';` and `::NEGATIVE = 'falsely';` it could make sense - let's keep it as is